src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/EmptyLineSeparatorCheck.java

```diff
@@ -46,6 +46,18 @@
  * is required at its end (for example for CLASS_DEF it is after '}').
  * Also, trailing comments are skipped.
  * </p>
+ * <p>
+ * ATTENTION: violations from multiple empty lines cannot be suppressed via XPath (#8179).
+ * Example:
+ * </p>
+ * <pre>
+ * class Test {
+ *     private int k;
+ *
+ *                                  // violation, multiple empty lines
+ *     private static void foo() {}
+ * }
+ * </pre>
```

Comment:
it is unavoidable that we will get RegexpMultiline violation when we want to show a example about multiple empty lines. So I find a way to add a comment on multiple empty lines, is it ok?

Reply:
please print config config also to show how exactly to reproduce this problem.
I tried your input no vioaltion.

```
rivanov@p5510:/var/tmp$ cat config.xml 
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
          "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
          "https://checkstyle.org/dtds/configuration_1_3.dtd">

<module name="Checker">
    <module name="TreeWalker">
      <module name="EmptyLineSeparator">
      </module>
    </module>
</module>
rivanov@p5510:/var/tmp$ cat Test.java 
class Test {
    private int k;


    private static void foo() {}
}
rivanov@p5510:/var/tmp$ java -jar /var/tmp/checkstyle-8.32-all.jar -c config.xml Test.java 
Starting audit...
Audit done.
```

after update please share CLI output of your example and in issue description and screenshot of web site after generation.

Reply:
done, comment link is here https://github.com/checkstyle/checkstyle/issues/7730#issuecomment-638239983

Reply:
violation is not on line where you put `//violation`, violation is on line 6,checkstyle report it on 7, that, please clarify, Lets keep in descrition `ATTENTION: violations from multiple empty lines cannot be suppressed via XPath (until #8179).` Please move example of code and config to "Example" section.

Reply:
yes, I know that violation is not on line where I put, reason is that it seems unavoidable that we will get RegexpMultiline violation when we want to show an example about multiple empty lines. So I find this way to add a comment on multiple empty lines but which cause another problem is that violation is not on line where I put.

Reply:
@HuGanghui , please do what I requested in https://github.com/checkstyle/checkstyle/pull/8296#discussion_r434665717 

please show CLI output in PR description for newly added Example, please place `//violation` on line where violation is placed by checkstyle.  We can do `// violation, before his like there two empty lines`

Reply:
done

Reply:
@HuGanghui @romani As far as I understood we log violations about empty lines based on AST element following or preceding  empty line.

What if [here](https://github.com/checkstyle/checkstyle/blob/master/src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/EmptyLineSeparatorCheck.java#L423), instead of logging line number we log nearest following AST element?

What do you think guys?

Reply:
@timurt , this exactly what https://github.com/checkstyle/checkstyle/issues/8179 is about.

Reply:
@HuGanghui , please explain why we can not suppress violation that you reported in PR description by Xpath, it is reported on AST node, it is suppress-able. Please read #8179 attentively, there are problems with lines that comes after method definition.